src/App.js

```diff
@@ -1,25 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+/** @format */
+
+import logo from "./logo.svg";
+import "./App.css";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Survey from "./Survey";
+import SurveyQuestions from "./Survey/SurveyQuestions";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	return (
+		<Router>
+			<Routes>
+				<Route exact path="/" element={<Survey />} />
+				<Route exact path="/SurveyQuestions" element={<SurveyQuestions />} />
```

Comment:
URI has some naming conventions. Please refer: https://api.gov.au/standards/national_api_standards/naming-conventions.html#:~:text=URI%20Naming%20Conventions&text=the%20URI%20MUST%20be%20specified,part%20of%20the%20base%20URI and make changes accordingly.

Reply:
resolved